package.json

```diff
@@ -4658,6 +4659,11 @@
 				"command": "gitlens.views.tags.setShowAvatarsOff",
 				"title": "Hide Avatars",
 				"category": "GitLens"
+			},
+			{
+				"command": "gitlens.views.branches.copyCurrentBranchName",
```

Comment:
I would recommend changing this to `gitlens.copyCurrentBranch` -- since this shouldn't live inside the views, it is a global command in the command palette.

Reply:
👍 Thanks, that's a good point--I agree this should be global. Fixed the naming to `gitlens.copyCurrentBranch`.